tests/scripts/test_scripts.py

```diff
@@ -863,7 +863,7 @@ def _run_train_bc_for_test_analyze_imit(run_name, sacred_logs_dir, log_dir):
     ),
 )
 def test_analyze_imitation(tmpdir: str, run_names: List[str], run_sacred_fn):
-    sacred_logs_dir = tmpdir_path = pathlib.Path(tmpdir)
+    sacred_logs_dir = tmpdir_path = types.parse_path(tmpdir)
```

Comment:
What was wrong with just using `pathlib.Path` here given `tmpdir` is guaranteed to be a `str`?

Reply:
parse_path also makes it absolute wrt cwd.